packages/mongodb-memory-server/postinstall.js

```diff
@@ -18,11 +18,12 @@ function doesModuleExist(name) {
   }
 }
 
-const modulePath = '../mongodb-memory-server-core/lib/postinstall';
+const modulePath = '../mongodb-memory-server-core/lib/util/postinstall-helper';
```

Comment:
`mongodb-memory-server-core` in `dependncies` so I think that nodejs will correctly resolve `require('mongodb-memory-server-core/lib/util/postinstall-helper')` without relative import with doesModuleExist-check.

Reply:
true

Reply:
6dc6343f03cd2f811825049a11454fe43cfb9aee and 6dc3222055ba099cad7f872f9ac814c45720b6b3 should apply this

Reply:
i will revert 6dc6343f03cd2f811825049a11454fe43cfb9aee again, because from my testing (and test results) it is needed, because yarn will symlink the inter-workspace dependencies (and when doing an local install for testing both npm and yarn symlink)

Reply:
i will need to revert the other one too, because tests (install) are failing

Reply:
Agreed. 

Honestly, I'm shocked that it didn't resolve correctly

<img width="1181" alt="Screen Shot 2020-10-13 at 01 10 14" src="https://user-images.githubusercontent.com/1946920/95782159-18f08680-0cf1-11eb-81dc-235e9b8bad97.png">

Reply:
i thought too that it would resolve correctly, but it seems like it dosnt

-> but i think in the tests the problem is, that yarn tries to run "postinstall" which depends on that the code it compiled, which it isnt at that point, we could fix this with [`--ignore-scripts`](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-ignore-scripts), downside would be other packages scripts wouldnt be executed too

Reply:
I think that we should stay on the current workaround via importing by the relative path with doesModuleExist-check.